BootloaderCorePkg/Tools/BuildUtility.py

```diff
@@ -2,7 +2,7 @@
 ## @ BuildUtility.py
 # Build bootloader main script
 #
```

Comment:
Given we have different FSP for different SKU, it is not possible to use a single image to support different silicon SKU.
So there is no point to have a combine all the GPIO for different SKUs into one GPIO table.
Maybe we could let each SKU has a CfgDataDef.yaml (e.g. CfgDataDefRpls.yaml CfgDataDefRplp.yaml, etc.). This way we could make GPIO config simple.

Reply:
With this PR, the outcome CfgDataDef.yaml is specific for the platform of the specified board name. That is, this PR reuse the commom part of CfgDataDef.yaml, and include the GPIO part for specified board during build time. While it's true that having separate CfgDataDef.yamls will be a simpler design, also can avoid the issue on ConfigEditor.
We may need to have separate CfgDataDef.yaml as well as separate CfgData_PayloadSelection.yaml
Will create another pull request soon based on the idea :)

Reply:
Consider to have only three CfgDataDef.yaml for ADLN, ADLP, ADLS
e.g. ADLPS, RPLP could be able to share the same yaml with ADLP